docs/content/docs/guide/running.md

```diff
@@ -76,7 +77,8 @@ Code will not be triggered.
 The PipelineRun will always run in the namespace of the Repository CRD associated with the repo
 that generated the event.
 
-You can follow the execution of your PipelineRun with the [tkn pac](../cli/#install) cli :
+You can monitor the execution using the through the command line with the [tkn
```

Comment:
nit: either the `using the` or `through the`  needs to be removed

Reply:
your ight thanks!